_pages/events/learnathon-2018.md

```diff
@@ -109,7 +109,7 @@ The ImageJ Legacy Course covered the topics: images, tables, regions of interest
 ### Developing ImageJ Ops
 
 -   Clone https://github.com/imagej/tutorials (but you already have it, right?)
--   Import `maven-projects/create-a-new-op` in Eclipse.
+-   Import `java/howtos/src/main/java/howto/ops/CreateANewOp` in Eclipse.
```

Comment:
The project to import is `java/howtos`, so that's what you would import into Eclipse. You generally do not import specific `.java` files directly, but rather the folder containing the `pom.xml`, which is the project level.